.circleci/config.yml

```
@@ -114,6 +114,15 @@ commands:
       - run: curl -o /tmp/protoc.zip -L https://github.com/protocolbuffers/protobuf/releases/download/v3.12.1/protoc-3.12.1-linux-x86_64.zip
       - run: unzip /tmp/protoc.zip -d $HOME/.local
 
+  setup-go-intg-deps:
+    steps:
+      - install-protoc # install newer version of protoc into #HOME/.local/bin, since default is proto2
```

```suggestion
      - install-protoc # Install newer version of protoc into $HOME/.local/bin, since default is proto2.
```
Though I don't know that this comment is really necessary anyway.